quorum/templates/08-epirus-chain-explorer.yaml

```diff
@@ -0,0 +1,61 @@
+{{ $scope := . }}
+{{- if eq $scope.Values.epirus.enabled true }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: epirus-chain-explorer-deployment
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      name: epirus-chain-explorer-deployment
+  template:
+    metadata:
+      name:  epirus-chain-explorer-deployment
+      labels:
+        app: epirus
+        tier: backend
+        name: epirus-chain-explorer-deployment
+    spec:
+      securityContext:
+      containers:
+      - name: epirus-api
+        image: web3labs/epirus-free-api:latest
```

Comment:
I would rather not use latest here which cause problems if there is an update with breaking changes, Image tag/version should be a Helm value (applies to all images used in this file)

Reply:
Sadly for epirus-web, api and ingestion only the 'latest' tag is available. Changed the image tag for MongoDB to 4.4.5.